src/library.js

```diff
@@ -405,6 +405,13 @@ mergeInto(LibraryManager.library, {
     abort('Assertion failed: ' + UTF8ToString(condition) + ', at: ' + [filename ? UTF8ToString(filename) : 'unknown filename', line, func ? UTF8ToString(func) : 'unknown function']);
   },
 
+  // TODO: Remove this when Rust doesn't need it anymore.
+  // https://github.com/rust-lang/rust/pull/97888
+  __gxx_personality_v0: function() {
+    console.warn("__gxx_personality_v0 called", arguments);
+    abort("__gxx_personality_v0 called");
```

Comment:
I think the right place for this to live is probably somewhere in `libcxxabi`.   We currently don't compile `src/cxa_personality.cpp`, but maybe we should compile something like it?

At least it think it would make sense for this symbol to live in native code rather than being an import from JS.